horace_core/sqw/file_io/CollabDiagram.gv

```diff
@@ -1,4 +1,4 @@
-digraph CollabDiag {
+	/*edge [dir=back,arrowhead=odiamond];*/digraph CollabDiag {
```

Comment:
worrying that we're using a non-standard graphics type. Are we expecting users generally to have access to .gv apps?

Reply:
no -- it used by us to be able to rebuild the diagram (and by who understands) For all others the generated diagram (png) with the same name if provided nearby